src/Parsers/ExpressionListParsers.cpp

```diff
@@ -125,21 +125,19 @@ bool ParserUnionList::parseImpl(Pos & pos, ASTPtr & node, Expected & expected)
     return true;
 }
 
-static bool parseOperator(IParser::Pos & pos, const char * op, Expected & expected)
+static bool parseOperator(IParser::Pos & pos, const std::string_view op, Expected & expected)
```

Comment:
It's alright, I can get rid of it. The underlying notion is that parameters are much like local variables so that the ones needn't be modified should be const.